runtime/src/pcr.rs

```diff
@@ -0,0 +1,124 @@
+// Licensed under the Apache-2.0 license
+
+use crate::Drivers;
+use caliptra_common::mailbox_api::{
+    ExtendPcrReq, IncrementPcrResetCounterReq, MailboxResp, MailboxRespHeader, QuotePcrsReq,
+    QuotePcrsResp,
+};
+use caliptra_drivers::{
+    cprintln, hand_off::DataStore, CaliptraError, CaliptraResult, Ecc384PrivKeyIn, KeyReadArgs,
+    PcrBank, PcrId,
+};
+use zerocopy::{transmute, FromBytes};
+
+pub struct PcrResetCounter {
+    counter: [u32; 32],
```

Comment:
Can you pull this const 32 value from drivers/src/pcr_bank.rs? That way, if the size of the pcr bank ever changes, they wouldn't have to update this file.

Reply:
#1004